src/content.js

```diff
@@ -248,14 +248,13 @@ function addDeleteForkLink() {
 	const postMergeDescription = select('#partial-pull-merging .merge-branch-description');
 
 	if (postMergeDescription) {
-		const currentBranch = postMergeDescription.querySelector('.commit-ref.current-branch');
+		const currentBranch = postMergeDescription.querySelector('.commit-ref');
 		const forkPath = currentBranch ? currentBranch.title.split(':')[0] : null;
 
 		if (forkPath && forkPath !== repoUrl) {
 			postMergeDescription.append(
 				<p id="refined-github-delete-fork-link">
 					<a href={`/${forkPath}/settings`}>
-						{icons.fork()}
```

Comment:
Can we keep the fork icon? If the problem was the missing space, you can add it with `{' '}`

Reply:
Hmm I guess the reason why you removed it is that GH also removed their icon:

![](https://cloud.githubusercontent.com/assets/170270/13520281/b2c9335c-e211-11e5-9e36-b0f325166356.png)